src/routes/blog/content.ts

```diff
@@ -105,9 +106,16 @@ export const categories = Object.values(categoriesGlob).map((categoryList) => {
     };
 });
 
-export const getAllBlogEntriesWithAuthors = () => {
+export const normalizeCategory = (str: string) => str?.replace(/\s+/g, '-').toLowerCase();
```

Comment:
this may be a bit sketchy but good for now, we need a proper category map at some point I think :-)

Reply:
we definitely do, there's quite inconsistency for categories mentioned and used across posts.